rest/data/commit_queue_test.go

```diff
@@ -343,8 +347,10 @@ func (s *CommitQueueSuite) TestWritePatchInfo() {
 	`
 
 	s.NoError(writePatchInfo(patchDoc, patchSummaries, patchContents))
-	s.Len(patchDoc.Patches, 1)
+	s.Require().Len(patchDoc.Patches, 1)
```

Comment:
most of our codebase seems to use s.len() and it looks like s.require().len seems to do the same thing.
would it be possible for us to use one version across the codebase for consistency/readability?

Reply:
these do different things -- .require() errors if the line fails (which is helpful for length checks where we would have a nil pointer dereference on future lines), whereas otherwise we'll just continue and evaluate all lines, even if one fails. It would be ideal if we used Require everytime this could happen but I would say it's really not worth auditing the codebase for.